lib/eventum/class.session.php

```diff
@@ -108,8 +108,6 @@ public static function init($usr_id)
      */
     public static function verify($usr_id)
     {
-        session_start();
-
         // Don't check the IP of the session, since this caused problems for users that use a proxy farm that uses
```

Comment:
It was causing a warning if any other code initiated a session before it. My testing showed that it was not needed as Session::init() (which is called by Session::verify() ) checks if a session is active and if not activates it. However, in I will add a similar check here in case something I couldn't find was depending on it.